cms/djangoapps/contentstore/api/views/utils.py

```diff
@@ -135,3 +138,19 @@ def _wrapper_view(self, request, course_id, *args, **kwargs):
             )
         return view(self, request, course_key, *args, **kwargs)
     return _wrapper_view
+
+
+class NumericalInputValidationView(GenericAPIView):
+    """Class in charge of NumericalInputValidations"""
+    def post(self, request):
+        """function to validate a math expression (formula) and return of the numeric input is valid or not"""
+        result = {'preview': '',
+                  'is_valid': True,
+                  'error': ''}
```

Comment:
Could you define serializer classes for the API input and output? These come with built-in validation and will make the API docs better.

I think something like this would work (haven't tested, but you can poke around edx-platform for other examples):
```python
from rest_framework import serializers

class NumericalInputValidationRequestSerializer(serializers.Serializer):
    formula = serializers.CharField()

class NumericalInputValidationReponseSerializer(serializers.Serializer):
    preview = serializers.CharField()
    is_valid = serializers.BooleanField()
    error = serializers.CharField(allow_null=True)
```